lib/src/main/java/com/orange/ods/compose/component/imagetile/OdsImageTile.kt

```diff
@@ -138,33 +133,71 @@ fun OdsImageItem(
                     }
                 }
 
-            OdsImageItemTitleType.None ->
-                Image(
-                    painter = image,
-                    contentDescription = imageContentDescription,
-                    contentScale = ContentScale.Crop,
-                    modifier = Modifier
-                        .fillMaxSize()
-                )
+            OdsImageTileCaptionDisplayType.None ->
+                image.Content(modifier = Modifier.fillMaxSize())
         }
     }
 }
 
-enum class OdsImageItemTitleType {
+/**
+ * An image in an [OdsImageTile].
+ */
+class OdsImageTileImage : OdsComponentImage {
+
+    /**
+     * Creates an instance of [OdsImageTileImage].
+     *
+     * @param painter The painter to draw.
+     * @param contentDescription The content description associated to this [OdsImageTileImage].
+     * @param contentScale The rule to apply to scale the image in this [OdsImageTileImage], [ContentScale.Crop] by default.
+     */
+    constructor(painter: Painter, contentDescription: String, contentScale: ContentScale = ContentScale.Crop) : super(
+        painter as Any,
+        contentDescription,
+        contentScale
+    )
+
+    /**
+     * Creates an instance of [OdsImageTileImage].
+     *
+     * @param imageVector The image vector to draw.
+     * @param contentDescription The content description associated to this [OdsImageTileImage].
+     * @param contentScale The rule to apply to scale the image in this [OdsImageTileImage], [ContentScale.Crop] by default.
+     */
+    constructor(imageVector: ImageVector, contentDescription: String, contentScale: ContentScale = ContentScale.Crop) : super(
+        imageVector as Any,
+        contentDescription,
+        contentScale
+    )
+
+    /**
+     * Creates an instance of [OdsImageTileImage].
+     *
+     * @param bitmap The image bitmap to draw.
+     * @param contentDescription The content description associated to this [OdsImageTileImage].
+     * @param contentScale The rule to apply to scale the image in this [OdsImageTileImage], [ContentScale.Crop] by default.
+     */
+    constructor(bitmap: ImageBitmap, contentDescription: String, contentScale: ContentScale = ContentScale.Crop) : super(
+        bitmap as Any,
+        contentDescription,
+        contentScale
+    )
+}
+
+enum class OdsImageTileCaptionDisplayType {
```

Comment:
I don't know if it's clear that the caption encapsulate both the title and the icon. On the other hand I can't find a better name than caption. Banner is the only one that comes to my mind and I am not sure it's better.

Reply:
I agree that "caption" may be a bit confusing cause we use this term for text style. I rename it to "legend area".